addons/im_livechat/static/src/new/chatbot/chatbot_service.js

```diff
@@ -0,0 +1,335 @@
+/* @odoo-module */
+
+import { ChatbotStep } from "@im_livechat/new/chatbot/chatbot_step_model";
+import { Chatbot } from "@im_livechat/new/chatbot/chatbot_model";
+import { SESSION_STATE } from "@im_livechat/new/core/livechat_service";
+
+import { EventBus, reactive } from "@odoo/owl";
+
+import { _t } from "@web/core/l10n/translation";
+import { browser } from "@web/core/browser/browser";
+import { debounce } from "@web/core/utils/timing";
+import { registry } from "@web/core/registry";
+
+export class ChatBotService {
+    static MESSAGE_DELAY = 2000;
+    // Time between two messages coming from the bot.
+    static STEP_DELAY = 500;
+    // Time to wait without user input before considering a multi line
+    // step as completed.
+    static MULTILINE_STEP_DEBOUNCE_DELAY = 10000;
```

Comment:
All these `static` can be `const`.
This would make code slightly more readable, by not having `ChatbotService.` as prefix to get these constants.